gobusterdir/http.go

```diff
@@ -68,9 +68,14 @@ func newHTTPClient(c context.Context, opt *OptionsDir) (*httpClient, error) {
 }
 
 // MakeRequest makes a request to the specified url
-func (client *httpClient) makeRequest(fullURL, cookie string) (*int, *int64, error) {
+func (client *httpClient) makeRequest(fullURL, cookie string, opt *OptionsDir) (*int, *int64, error) {
```

Comment:
If you’re passing in the full options now, I wonder if it makes sense to pull the cookie out of this function params, and just extract it from the options inside this function as needed

Reply:
+1 to this. Pass in all options instead.

Reply:
Indeed it seemed redundant to leave the cookie parameter when it was already being passed in along with the options. I was going for a light touch on the codebase, hence the initial decision to simply add one parameter.